<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Bot Race</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> QMainWindow::separator {
     width: 0px; /* when vertical */
     height: 0px; /* when horizontal */
 }</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Game">
    <property name="title">
     <string comment="show game menu">&amp;Game</string>
    </property>
    <addaction name="action_New_Game"/>
    <addaction name="action_Join_Game"/>
    <addaction name="action_Stop_Game"/>
    <addaction name="separator"/>
    <addaction name="actionHost_Game"/>
    <addaction name="actionLeave_Game"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string comment="Show help menu">&amp;Help</string>
    </property>
    <addaction name="actionShow_Simulator"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_BotRace"/>
    <addaction name="separator"/>
    <addaction name="actionManual"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string comment="show settings menu">&amp;Settings</string>
    </property>
    <widget class="QMenu" name="menuGUI_size">
     <property name="title">
      <string comment="select the gui size">GUI size</string>
     </property>
     <addaction name="actionSmall"/>
     <addaction name="actionMedium"/>
     <addaction name="actionBig"/>
    </widget>
    <widget class="QMenu" name="menuAnimation_Speed">
     <property name="title">
      <string comment="select animation speed">Animation Speed</string>
     </property>
     <addaction name="actionSlow"/>
     <addaction name="actionNormal"/>
     <addaction name="actionFast"/>
    </widget>
    <addaction name="menuGUI_size"/>
    <addaction name="actionFixed_Layout"/>
    <addaction name="separator"/>
    <addaction name="menuAnimation_Speed"/>
    <addaction name="separator"/>
    <addaction name="actionGame_Settings"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionZoom_fit_to_view"/>
    <addaction name="separator"/>
    <addaction name="actionCenter_on_player"/>
   </widget>
   <addaction name="menu_Game"/>
   <addaction name="menuView"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_New_Game">
   <property name="icon">
    <iconset resource="../../icons/gameicons.qrc">
     <normaloff>:/icons/arrow-right.png</normaloff>:/icons/arrow-right.png</iconset>
   </property>
   <property name="text">
    <string comment="Starts a new loal game">&amp;New Game</string>
   </property>
  </action>
  <action name="action_Join_Game">
   <property name="icon">
    <iconset resource="../../icons/gameicons.qrc">
     <normaloff>:/icons/network-connect.png</normaloff>:/icons/network-connect.png</iconset>
   </property>
   <property name="text">
    <string comment="join a network game">&amp;Join Game</string>
   </property>
  </action>
  <action name="action_Stop_Game">
   <property name="icon">
    <iconset resource="../../icons/gameicons.qrc">
     <normaloff>:/icons/list-remove.png</normaloff>:/icons/list-remove.png</iconset>
   </property>
   <property name="text">
    <string comment="stop the local game">&amp;Stop Game</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="../../icons/gameicons.qrc">
     <normaloff>:/icons/application-exit.png</normaloff>:/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string comment="quit game">&amp;Exit</string>
   </property>
  </action>
  <action name="actionSounds">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string comment="activate sound">Sounds</string>
   </property>
  </action>
  <action name="actionSmall">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string comment="small gui element size">Small</string>
   </property>
  </action>
  <action name="actionMedium">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string comment="medium gui element size">Medium</string>
   </property>
  </action>
  <action name="actionBig">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string comment="big gui element size">Big</string>
   </property>
  </action>
  <action name="actionSlow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string comment="slow animation speed">Slow</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string comment="normal animation speed">Normal</string>
   </property>
  </action>
  <action name="actionFast">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string comment="fast animation speed">Fast</string>
   </property>
  </action>
  <action name="actionGame_Settings">
   <property name="icon">
    <iconset resource="../../icons/gameicons.qrc">
     <normaloff>:/icons/configure.png</normaloff>:/icons/configure.png</iconset>
   </property>
   <property name="text">
    <string comment="show game settings">Game Settings</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string comment="show about qt dialog">About Qt</string>
   </property>
  </action>
  <action name="actionAbout_BotRace">
   <property name="icon">
    <iconset resource="../../icons/gameicons.qrc">
     <normaloff>:/icons/help-about.png</normaloff>:/icons/help-about.png</iconset>
   </property>
   <property name="text">
    <string comment="show about botrace dialog">About BotRace</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="icon">
    <iconset resource="../../icons/gameicons.qrc">
     <normaloff>:/icons/help-contents.png</normaloff>:/icons/help-contents.png</iconset>
   </property>
   <property name="text">
    <string comment="show the game  manual">Manual</string>
   </property>
  </action>
  <action name="actionFixed_Layout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string comment="allow to chnage gui elements or not">Fixed Layout</string>
   </property>
  </action>
  <action name="actionHost_Game">
   <property name="icon">
    <iconset resource="../../icons/gameicons.qrc">
     <normaloff>:/icons/application-x-smb-server.png</normaloff>:/icons/application-x-smb-server.png</iconset>
   </property>
   <property name="text">
    <string comment="host a newtwork game">Host Game</string>
   </property>
  </action>
  <action name="actionLeave_Game">
   <property name="icon">
    <iconset resource="../../icons/gameicons.qrc">
     <normaloff>:/icons/network-disconnect.png</normaloff>:/icons/network-disconnect.png</iconset>
   </property>
   <property name="text">
    <string comment="leave a network game">Leave Game</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionZoom_fit_to_view">
   <property name="text">
    <string>Zoom fit to view</string>
   </property>
  </action>
  <action name="actionCenter_on_player">
   <property name="text">
    <string>Center on player</string>
   </property>
  </action>
  <action name="actionShow_Simulator">
   <property name="text">
    <string>Show Simulator</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../icons/gameicons.qrc"/>
  <include location="../../icons/gameicons.qrc"/>
  <include location="../../icons/gameicons.qrc"/>
 </resources>
 <connections/>
</ui>
